src/components/DuosHeader.js

```diff
@@ -161,6 +170,17 @@ class DuosHeader extends Component {
     });
   };
 
+  makeNotifications = () => {
+    const dataList = [];
+    const data = this.state.notificationData;
+    for (let i in data) {
+      if (data[parseInt(i)].hasOwnProperty("id")) {
+        dataList.push(data[parseInt(i)]);
+      }
+    }
+    return dataList.map((d) => Notification({notificationData: d}));
+  };
```

Comment:
Whenever you find yourself iterating over lists and using temporary holding variables, there is almost always a cleaner, functional way to do it with filters and maps. For example, I started with this:
```
...
import { get, filter, flow, map } from 'lodash/fp';
...
    const data = this.state.notificationData;
    const dataList = filter(get('id'))(data);
    return map((d) => Notification({notificationData: d}))(dataList);
```
which breaks out the three main concepts in this method,
* get the data
* filter on those with an id
* map those to a new notification

Once we have that, we can simplify even further:
```
    const dataList = filter(get('id'))(this.state.notificationData);
    return map((d) => Notification({notificationData: d}))(dataList);
```
And then reducing again:
```
    return map((d) => Notification({notificationData: d}))(filter(get('id'))(this.state.notificationData));
```

That gets a little hard to read, but it's a one-liner with no temporary variables. Another way to clean this up is to use lodash flow which is a way to chain functions together, one at a time, passing the result of the first to the next, etc., which is what we're doing in that one-liner:
```
    return flow(
      filter(get('id')),
      map((d) => Notification({notificationData: d}))
    )(this.state.notificationData);
```

TBH, I don't think we need the filter line, but I think it's OK for safety.